root/account/Preferences.js

```diff
@@ -17,10 +17,10 @@ import type {PreferencesFormPropsT}
   from '../static/scripts/account/components/PreferencesForm';
 import * as manifest from '../static/manifest';
 
-type Props = {|
-  +$c: {user: EditorT} & CatalystContextT,
+type Props = {
+  +$c: {...CatalystContextT, user: EditorT},
```

Comment:
Both `{user: EditorT} & CatalystContextT` and `{...CatalystContextT, user: EditorT}` should work (both giving exact types), but we tend to use spreads more, so I'm just standardizing on that syntax.